tools/carveme/carveme.xml

```diff
@@ -0,0 +1,107 @@
+<tool id="carveme" name="CarveMe" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>Tool to reconstruct genome-scale metabolic models</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.6.0</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+        <xml name="biotools">
+            <xrefs>
+                <xref type="bio.tools">CarveMe</xref>
+            </xrefs>
+        </xml>
+    </macros>
+    <requirements>
+        <requirement type='package' version="@TOOL_VERSION@">carveme</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    carve '$annotation'
+    #if $dna
+        --dna
+    #end if
+    #if $egg
+        --egg 
+    #end if
+    #if $fc2
+        --fc2 
+    #end if
+    #if $cobra
+        --cobra 
+    #end if
+    #if $diamond_args
+        --diamond-args '$diamond_args'
+    #end if
+    #if $universe
+        --universe '$universe'
+    #end if 
+    #if $universe_file
+        --universe-file '$universe_file'
+    #end if
+    #if $mediadb
+        --mediadb '$mediadb'
+    #end if
+    #if $soft
+        --soft '$soft'
+    #end if
+    #if $hard
+        --hard '$hard'
+    #end if
+    #if $reference
+        --reference '$reference'
+    #end if
+    --solver scip
+    --output '$model'
+    ]]></command>
+    <inputs>
+    <param name="annotation" label="Annotation file" type="data" format="text" optional="false" multiple="false"/>
+    <param argument="--dna" label="Build from DNA FASTA file" type="boolean" truevalue="--dna" falsevalue="" optional="true"/>
+    <param argument="--egg" label="Build from eggNOG-mapper annotation file" type="boolean" truevalue="--egg" falsevalue="" optional="true"/>
+    <param argument="--fc2" label="Output SBML in sbml-fbc2 format" type="boolean" truevalue="--fc2" falsevalue="" optional="true"/>
+    <param argument="--universe" label="Pre-built universe model (default: Bacteria)" type="select" optional="true" multiple="false">
+        <option value="grampos">Gram Positive</option>
+        <option value="gramneg">Gram Negative</option>
+        <option value="cyanobacteria">Cyanobacteria</option>
+        <option value="archaea">Archaea</option>
+    </param>
+    <param argument="--cobra" label="Output SBML in old cobra format" type="boolean" truevalue="--cobra" falsevalue="" optional="true"/>
+    <param argument="--diamond_args" label="Additional arguments for running diamond"  type="text" optional="true"/>
+    <param argument="--universe_file" label="Reaction universe file (SBML format)"  type="data" format="sbml" optional="true"/>
```

Comment:
```suggestion
    <param argument="--universe-file" label="Reaction universe file (SBML format)"  type="data" format="sbml" optional="true"/>
```

It seems the original argument is with `-`? That is ok, Galaxy will convert `-` to `_` to generate the variable name.